PhysicsTools/PatUtils/interface/MiniIsolation.h

```diff
@@ -0,0 +1,35 @@
+#ifndef MiniIsolation_h
+#define MiniIsolation_h
+
+/*
+
+  Defines a MiniIsolation struct and a function to
+  compute MiniIsolation given a 4-vector and a collection
+  of packed PF candidates
+
+*/
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+
+namespace pat{
+
+    struct MiniIsolation {
+        float chiso; //charged hadrons
+        float nhiso; //neutral hadrons
+        float phiso; //photons
+        float puiso; //pileup
+    };
+
+    typedef ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > LorentzVector;
+
+    MiniIsolation GetMiniPFIsolation(const pat::PackedCandidateCollection *pfcands, LorentzVector p4,
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.